examples/bods/unflatten/expected/out.json

```diff
@@ -1,24 +1,24 @@
 [
     {
```

Comment:
I don't like the indentation mismatch here between this opening `{` and the closing one. All the other examples look okay, so I think this has to do with having a list at the root.

Reply:
Looks like this is a `jsonstreams` bug, I can reproduce with:

```
import jsonstreams

with jsonstreams.Stream(jsonstreams.Type.array, filename='test.json', indent=4) as s:
    s.write({"a": 1, "b": 2})
```

I will report this upstream.

Reply:
Reported https://github.com/dcbaker/jsonstreams/issues/41